build/gulpfile.extensions.js

```diff
@@ -25,6 +25,12 @@ const _ = require('underscore');
 
 const extensionsPath = path.join(path.dirname(__dirname), 'extensions');
 
+/**Use of 'debug' allows one to emit console (error stream by default) on demand.
+ *  By default nothing is emitted unless environment variable DEBUG is defined.
+ * @see https://www.npmjs.com/package/debug
```

Comment:
I'm assuming all this tracing was for development? This should be removed.

Reply:
Nope - it is turned off by default so I plan to leave this in for any future developer in need of similar tracing.

Reply:
I very much disagree with that, this make merge conflicts with vscode way more likely, please remove.

Reply:
removed all tracing. There is still a bug fix to honor outDir parameter of tsconfig.json which this gulpfile was ignoring.